src/behaviors/toggle/index.js

```diff
@@ -0,0 +1,84 @@
+// @flow
```

Comment:
Other core components and behaviors are prefixed with `hv-` (directory) / `Hv` (when imported) - could you adjust this?